src/logics/Logic.cc

```diff
@@ -1270,6 +1270,99 @@ Logic::dumpChecksatToFile(ostream& dump_out) const
     dump_out << "(exit)" << endl;
 }
 
+string
+Logic::printString(PTRef formula, bool negate, bool toassert) const
```

Comment:
This method needs some love.
Some style and formatting cleanup.
Also, don't we have something similar already?

Reply:
Sure, done

Reply:
BTW, can't we use printTerm:
https://github.com/MasoudAsadzade/opensmt-verification/blob/7f119d1ba837216e42aa951891ffe79328e08670/src/logics/Logic.cc#L232
for printing partition and actual clauses ?